spec/directives/mixin/content/arguments/none.hrx

```diff
@@ -0,0 +1,105 @@
+<===> no_parens/no_using/input.scss
+@mixin mixin {
+  @content;
+}
+
+a {
+  @include mixin {
+    x: y;
+  }
+}
+
+<===> no_parens/no_using/output.css
+a {
+  x: y;
+}
```

Comment:
It seems strange to have a spec under `content/arguments` that calls a mixin without passing any arguments to its content block.